src/MockClassifier.Api/appsettings.json

```diff
@@ -7,7 +7,9 @@
   },
   "AllowedHosts": "*",
   "DmrServiceSettings": {
-    "DmrApiUri": "https://tablelogapi.azurewebsites.net/app/fromclassifier",
-    "DmrRequestProcessIntervalMs": 5000
+    "DmrServiceSettings": {
+      "CentOpsUri": "http://127.0.0.1:8080",
+      "CentOpsApiKey": "<generate a key>"
+    }
```

Comment:
This doesn't seem to match the model of the settings object.   Is this right or am I missing something?